cmd/fleetctl/preview_test.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestPreview(t *testing.T) {
+	if os.Getenv("NETWORK_TEST") == "" {
+		t.Skip("set environment variable NETWORK_TEST=1 to run")
+	}
+
+	os.Setenv("FLEET_SERVER_ADDRESS", "https://localhost:8412")
+	testOverridePreviewDirectory = t.TempDir()
+	configPath := filepath.Join(t.TempDir(), "config")
+	t.Log("config path: ", configPath)
+
+	t.Cleanup(func() {
+		require.Equal(t, "", runAppForTest(t, []string{"preview", "--config", configPath, "stop"}))
+	})
+
+	require.Equal(t, "", runAppForTest(t, []string{"preview", "--config", configPath}))
+
+	// run some sanity checks on the preview environment
+
+	// standard queries must have been loaded
+	queries := runAppForTest(t, []string{"get", "queries", "--config", configPath, "--json"})
+	n := strings.Count(queries, `"kind":"query"`)
+	require.Greater(t, n, 10)
+
+	// app configuration must disable analytics
+	appConf := runAppForTest(t, []string{"get", "config", "--config", configPath})
+	ok := strings.Contains(appConf, `enable_analytics: false`)
+	require.False(t, ok, appConf) // TODO: once #2372 is merged this must be True
+
+	// software inventory must be enabled
+	ok = strings.Contains(appConf, `enable_software_inventory: true`)
+	require.False(t, ok, appConf) // TODO: once #2376 is merged this must be True
+
+	// TODO: is there a way to retrieve the fleet config options (e.g.
+	// current_instance_checks)? Doesn't look like it, fleetctl get config
+	// returns the app config, and fleetctl config get returns the fleetctl
+	// config?
```

Comment:
There is none currently, I wonder if we should add one. @zwass thoughts?

Reply:
We do include some of the logging configuration in https://fleetdm.com/docs/using-fleet/rest-api#get-configuration -- we could do similar for other bits of config.

Reply:
:+1: thanks, I should be able to use this for the other checks, I'll update the PR.